pkg/config/configsections/deployment.go

```diff
@@ -16,10 +16,18 @@
 
 package configsections
 
-// Deployment defines a deployment in the cluster.
-type Deployment struct {
+// PodSet defines a deployment /Statefullset in the cluster.
+type PodSet struct {
 	Name      string
 	Namespace string
 	Replicas  int
 	Hpa       Hpa
+	Type      PodSetType
 }
+
+type PodSetType string
+
+const (
+	Deployment   PodSetType = "deployment"
```

Comment:
you already defined this in deployment_info. Is there a way to keep a single definition only and import it when needed ?